File: scripts/GAICL/Experiment_Results_v0-Python-code/intermediate/SR/Translation/use_test/deepseek-coder-33b-instruct/Avatar/Python/Java/atcoder_ABC112_A.java
Here is the Java equivalent of the Python code:

```java
import java.util.Scanner;

public class Main {
    static int condition_1 = 836;
    static int condition_2 = 829;

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int age = scanner.nextInt();

        if ((condition_1 & condition_2) != 0) {
            if (age == 1) {
                System.out.println("Hello World");
            } else {
                int input_a = scanner.nextInt();
                int input_b = scanner.nextInt();
                System.out.println(sumInputs(input_a, input_b));
            }
        }
    }

    public static int sumInputs(int a, int input_b) {
        return a + input_b;
    }
}
```

Please note that Java does not have a direct equivalent to Python's `shuffle` function from `sklearn.utils` or `ttest_ind` function from `scipy.stats`. These functions are used for data manipulation and statistical analysis, respectively, and are not directly available in Java.

Also, the `my_decorator` function in the Python code is not used in the provided code, so it is not included in the Java translation.

File: dataset/Intermediate/Translation/Avatar/v0-Python-code/atcoder_ABC112_A.py
condition_1 = 836
condition_2 = 829

def my_decorator(func):

    def dec_result(*args, **kwargs):
        result = func(*args, **kwargs)
        return result
    return dec_result

@my_decorator
def sum_inputs(a, input_b):
    return a + input_b
from sklearn.utils import shuffle
shuffle([24, 34, 75])
from scipy.stats import ttest_ind
ttest_ind([93, 20, 18], [38, 3, 91])
age = int(input())
if condition_1 & condition_2:
    if age == 1:
        print('Hello World')
    else:
        (input_a, input_b) = [int(input()) for counter in range(2)]
        print(sum_inputs(input_a, input_b))